.github/workflows/pytest.yml

```diff
@@ -50,11 +50,11 @@ jobs:
           pip install --no-deps --editable .
           make docs-build
 
-      - name: Upload coverage
+      - name: Upload docs coverage artifact
         uses: actions/upload-artifact@v4
         with:
           name: coverage-docs
-          path: coverage.xml
+          path: ./*coverage*
```

Comment:
Upload both the `coverage.xml` report and the `.coverage` outputs so they're both available to the different coverage actions later.